experimental/generation/generator/packages/library/templates/standard/dynamiclist.schema

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "string",
+    "$generator": {
```

Comment:
I thought you needed $dynamicList: true here as well?

Reply:
You should also have a $template: "dynamicList" and a corresponding dynamicList.lg file which has the $templates you need to expand this.

---
In reply to: [609963431](https://github.com/microsoft/BotBuilder-Samples/pull/3150#discussion_r609963431) [](ancestors = 609963431)

Reply:
I think we do not need $template in dynamiclist.schema .

Reply:
That is because you are making the type enum and not string.  

---
In reply to: [610484318](https://github.com/microsoft/BotBuilder-Samples/pull/3150#discussion_r610484318) [](ancestors = 610484318)